lisp/ess-custom.el

```diff
@@ -2587,11 +2587,11 @@ This variable has no effect. Customize
   (append ess-RS-constants
           '("T" "F")))
 
-(defconst ess-r--keywords
+(defvar ess-r--keywords
   '("in" "else" "break" "next"))
 (defvaralias 'ess-R-keywords 'ess-r--keywords)
 
-(defconst ess-r--fn-like-keywords
+(defvar ess-r--fn-like-keywords
```

Comment:
`defconst` is really not a const in emacs:)

`--` is really not warranted here. It's a kinda user level variable even if it's not a custom.

Reply:
I think these variables should be modifiable (I have now switched it back to defvar for this reason) but that they should not be part of a configuration UI. I.e. they should not constrain possible updates to how we fontify things.

I have reverted the naming back to the old scheme though so the `--` is gone.